src/main/java/io/github/thebusybiscuit/slimefun4/core/attributes/PiglinBarterDrop.java

```diff
@@ -0,0 +1,39 @@
+package io.github.thebusybiscuit.slimefun4.core.attributes;
+
+import org.bukkit.entity.EntityType;
+import org.bukkit.entity.Player;
+import org.bukkit.event.entity.EntityDeathEvent;
+import org.bukkit.event.entity.EntityDropItemEvent;
+
+import io.github.thebusybiscuit.slimefun4.implementation.items.misc.BasicCircuitBoard;
+import io.github.thebusybiscuit.slimefun4.implementation.listeners.MobDropListener;
+import me.mrCookieSlime.Slimefun.Lists.RecipeType;
+import me.mrCookieSlime.Slimefun.Objects.SlimefunItem.SlimefunItem;
+
+/**
+ * This interface, when attached to a {@link SlimefunItem}, provides an easy method for adding
+ * a % chance to drop for an {@link SlimefunItem} on {@link EntityItemDropEvent}, this chance is 0-100
```

Comment:
"provides an easy method for adding a % chance to drop for an SlimefunItem"
This doesn't make sense. Also this is one entire sentence. That's a long ass sentence. Also, grammar pls